bindings/ruby/bin/rock-gazebo-viz

```diff
@@ -54,7 +59,19 @@ sdf.each_model(recursive: true) do |model|
     model_viz.frame = model.name
 
     task_name = "gazebo:#{model.full_name.gsub('::', ':')}"
-    task_proxy = Orocos::Async.proxy task_name
```

Comment:
Ok Sylvain, I got it. 
I will close the PR and delete the branch. After I will change the startup script to apply the configuration.